Homework_PIP_Virtualenv_Virtualenvwrapper/task5.py

```diff
@@ -1,17 +1,15 @@
-# Write a function that prints numbers from 1 to 30. But for multiples of three print “Fizz” instead of the number and for the multiples of five print “Buzz”. For numbers which are multiples of both three and five print “FizzBuzz”.
+def func_fizz_buzz():
+    list = []
+    for i in range(1, 31):
+        if i % 3 == 0 and i % 5 == 0:
+            list.append("FizzBuzz")
+        elif i % 3 == 0:
+            list.append("Fizz")
+        elif i % 5 == 0:
+            list.append("Buzz")
+        else:
+            list.append(i)
```

Comment:
you can change some value and append only once in the end of loop
f.ex:
default_value = i
if i % 3 == 0 and i % 5 == 0:
    default_value = 'FizzBuzz'
.....

my_list.append(default_value)